### DISCUSSION TITLE
This order cannot be cancelled

### DISCUSSION THREAD

**Discussion by Megidd** [saleor/saleor#7828] [Q&A] [answered]
When I try to cancel a default Saleor order (created when setting up Saleor), I receive the following error. I appreciate any hint. Thanks! :smiley: 

![Screenshot_20210812_225953](https://user-images.githubusercontent.com/17475482/129250286-377fff3a-653d-473c-961e-51d9ff14fca3.png)

------------------

# Update

Before trying to cancel default orders, I deleted all default Saleor customers. Could it be the cause? I mean, maybe the customer cannot be found in database (and that's the cause)?

--------------

# Update

I'm using Saleor 2.11 :heavy_check_mark:

**Comment by karolkielecki** [comment, marked as answer]
@Megidd Hi, each order has to be related to a customer. So maybe that could be the cause. 

BTW, I recommend you using Saleor 3.0--we've released beta and it introduces many new features (multichannel and more) and improvements.
